pkgs/by-name/po/pokemon-colorscripts/package.nix

```
@@ -0,0 +1,39 @@
+{
+  stdenv,
+  fetchFromGitLab,
+  lib,
+  ...
+}:
+
+stdenv.mkDerivation {
+  pname = "pokemon-colorscripts";
+  version = "0-unstable-2024-10-19";
+
+  src = fetchFromGitLab {
+    owner = "phoneybadger";
+    repo = "pokemon-colorscripts";
+    rev = "5802ff67520be2ff6117a0abc78a08501f6252ad";
+    hash = "sha256-gKVmpHKt7S2XhSxLDzbIHTjJMoiIk69Fch202FZffqU=";
+  };
+
+  postPatch = ''
+    patchShebangs ./install.sh
```

```suggestion
    patchShebangs --build ./install.sh
```
A bit more clear in case someone wants to cross-build this